### ISSUE TITLE
Can't seem to get it to work in 1.21.0

### ISSUE THREAD

**Issue by Itja** [ErisApps/HitScoreVisualizer#16]
Hi, thanks for maintaining this awesome mod!

I have used it successfully in the past (i.e. several Beat Saber versions ago). However, I can't get it to work on a new installation of 1.21.0. The settings work correctly, I can see the menu where you can select the different available JSON files and could verify that it indeed loads these JSON files (by breaking my JSON config deliberately to see if it shows an error, which it did). Afterwards, I loaded a JSON config that should work (without any error showing up).

However, the score numbers still are the plain Beat Saber score numbers. 

How would I go forward to debug this? I have profound knowledge about C# and Visual Studio available, but did no Beat Saber modding so far, so I would be glad about some pointers on where to look.

**Comment by ErisApps** [comment]
Mind attaching your HSV config file so I can debug it myself?
I use Jetbrains Rider myself so don't really know how to properly debug using VS either, so is kinda hard to give you pointers on that. 😅

**Comment by Itja** [comment]
Hey, thanks for the fast response. I only got around to try it out once more today. It doesn't seem to be an issue with that specific HSV config, as it doesn't work for the default one nor for one from 115 Guild either. I uploaded them here, but I don't think they will not work for you.

* [Default](https://gist.github.com/Itja/be954b01391a002bb85aafa100cec837)
* [Mine](https://gist.github.com/Itja/3bb681b84017be31e0ebe66f6b3cca46)
* [From 115 Guild](https://gist.github.com/Itja/2bb9caaebba5d2ad1a15bd0d605e04fb)

As I said, I can switch around these config files successfully ingame in the menu, but they have no effect on the numbers being shown. Are there known incompatibilities with other mods maybe?

[My Beat Saber log](https://gist.github.com/Itja/c420bc7bdf16134c21f6d98a037d1ec9) only shows the mod being loaded. Does it maybe print something in the log when its hooks have been called successfully that is missing in my log?

Thanks for your help!
